container/CrabV2/Auction/AuctionBody.tsx

```diff
@@ -138,6 +138,26 @@ const BidForm: React.FC = () => {
     }
   }, [auction.isSelling, oSqthBalance, qty, totalWeth, wethBalance])
 
+  const balance = React.useMemo(() => {
+    if (auction.isSelling) {
+      return convertBigNumber(wethBalance)
+    } else {
+      return convertBigNumber(oSqthBalance)
+    }
+  }, [auction.isSelling, oSqthBalance, wethBalance])
+
+  const balanceToken = React.useMemo(() => {
+    if (auction.isSelling) {
+      return "WETH"
+    } else {
+      return "oSQTH"
+    }
+  }, [auction.isSelling])
+  
+  const setMaxBalance = React.useCallback(async () => {
+    setQty(balance.toString())
```

Comment:
Quantity is always osqth value, 
If auction is buying, user enter sqth the sell
If auction is selling, user enters sqth to buy based on user's weth balance. So in this case we should use "weth balance / price" to get sqth amount

Reply:
Updated!